src/compiler/crystal/command.cr

```diff
@@ -39,6 +39,7 @@ class Crystal::Command
     Usage: crystal tool [tool] [switches] [program file] [--] [arguments]
 
     Tool:
+        apply-types              add type restrictions to all untyped defs and def arguments
```

Comment:
Name suggestions welcome! Another option I thought of is `apply-restrictions` as that feels more accurate, but also longer to type.

Reply:
`bake-types` may also work, `apply-restrictions` is also pretty good

Reply:
How about `typify`?

Reply:
I like it!